tools/python/translate.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+# Requires `brew install translate-shell`
+# and DEEPL_FREE_API_KEY or DEEPL_API_KEY environment variables set.
+
+import os
+import subprocess
+import requests
+import json
+import sys
+
+# Use DeepL when possible with a fall back to Google.
+GOOGLE_TARGET_LANGUAGES = [
+  'ar',
+  'be',
+  'ca',
+  'eu',
+  'fa',
+  'he',
+  'mr',
+  'th',
+  'vi',
+  'zh-TW',  # zh-Hant in OM
+]
+
+DEEPL_TARGET_LANGUAGES = [
+    'bg',
+    'cs',
+    'da',
+    'de',
+    'el',
+    'en-GB',
+    'en-US', # en in OM
+    'es',
+    'et',
+    'fi',
+    'fr',
+    'hu',
+    'id',
+    'it',
+    'ja',
+    'ko',
+#    'lt',
+#    'lv',
+    'nb',
+    'nl',
+    'pl',
+    'pt-BR',
+    'pt-PT',  # pt in OM
+    'ro',
+    'ru',
+    'sk',
+#    'sl',
+    'sv',
+    'tr',
+    'uk',
+    'zh',  # zh-Hans in OM
+]
+
+
+def get_api_key():
+  key = os.environ.get('DEEPL_FREE_API_KEY')
+  if key == None:
+    key = os.environ.get('DEEPL_API_KEY')
+  if key == None:
+    print('Error: DEEPL_FREE_API_KEY or DEEPL_API_KEY env variables are not set')
+    exit(1)
+  return key
+
+def google_translate(text):
+  fromTo = 'en:'
+  for lang in GOOGLE_TARGET_LANGUAGES:
+    fromTo += lang + '+'
+  # Remove last +
+  fromTo = fromTo[:-1]
+  res = subprocess.run(['trans', '-b', '-no-bidi', fromTo, text], text=True, capture_output=True)
```

Comment:
It would be better to call the API directly instead of relying on additional external tool.

Reply:
Then we need to solve additional issues with API keys. You can always improve this working version later if you have enough time.

Reply:
https://github.com/organicmaps/organicmaps/commit/c452ffb7857f4cfc457527bfee1eea46264755c8